ci/python-publish.yml

```diff
@@ -0,0 +1,27 @@
+name: Upload Python Package
+
+on: [release]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python
+      if: github.event.action == 'created'
+      uses: actions/setup-python@v1
+      with:
+        python-version: '3.x'
```

Comment:
This presumably needs to be configurable, right?

Reply:
Yeah, it is up to the user to change this when they generate the workflow from this template. I chose to set the default as Python 3 since Python 2 support is getting dropped soon.